### PR TITLE
Added a NSFW detector on the scraper side.

### PR DESCRIPTION
**Pull request by mehrdadhalali**

### CODE REVIEW

File: web_scraping/gog_scraper/scrape_gog_game.py [hunk 1]
```diff
@@ -73,6 +73,21 @@ def extract_data_from_rows(rows: list[str], label_class: str, required_label: st
...
 
+def has_nsfw_warning(game_page) -> bool:
+    """Does this game have a NSFW warning?"""
+
+    modules = game_page.find_all("p", {"class": "module"})
+
+    if len(modules) == 0:
+        return False
+
+    if any("not appropriate for all ages" in module.text
```
Comment (Leon-JS02) [inline web_scraping/gog_scraper/scrape_gog_game.py:84]: Shouldn't you be converting to lower before checking for this?

Comment (Leon-JS02) [reply web_scraping/gog_scraper/scrape_gog_game.py:84]: Nevermind, I just took a look at GOG's site.

**Review by Leon-JS02 — APPROVED** [review]
This should work well for filtering!

File: web_scraping/gog_scraper/scrape_gog_game.py [hunk 2]
```diff
@@ -94,6 +109,8 @@ def get_game_data_from_url(game_url: str) -> dict:
                                     label_class="details__category table__row-label",
                                     required_label="genre", are_links=True)
 
+    is_nsfw = has_nsfw_warning(soup)
```
Comment (paulauus) [inline web_scraping/gog_scraper/scrape_gog_game.py:112]: Soup!

**Review by paulauus — APPROVED** [review]

**Pull request merged**